spotipy/util.py

```diff
@@ -1,4 +1,4 @@
-""" Shows a user's playlists (need to be authenticated via oauth) """
+""" Shows a user's playlists. This needs to be authenticated via OAuth) """
```

Comment:
forgotten bracket
```suggestion
""" Shows a user's playlists. This needs to be authenticated via OAuth """
```

Reply:
I have removed the bracket and added a period instead.